master/buildbot/steps/source/git.py

```diff
@@ -45,35 +45,64 @@ def isTrueOrIsExactlyZero(v):
 
 git_describe_flags = [
     # on or off
-    ('all', lambda v: ['--all'] if v else None),
```

Comment:
This PR is almost unreviewable due to all the extra changes. Can they be reverted?